DropInDemo/Features/Drop In/BraintreeDemoDropInViewController.m

```diff
@@ -22,6 +22,7 @@ @interface BraintreeDemoDropInViewController () <PKPaymentAuthorizationViewContr
 @property (nonatomic, strong) UIButton *dropInButton;
 @property (nonatomic, strong) UIButton *purchaseButton;
 @property (nonatomic, strong) UISegmentedControl *dropinThemeSwitch;
+@property (nonatomic, strong) UISegmentedControl *threeDSecureSwitch;
```

Comment:
To be consistent with the Braintree Core SDK demo app, I feel that the toggle b/w 3DS 1, 3DS 2, and No 3DS should live in the settings on the Demo App instead of a segmented control.

Reply:
Addressed in new changes (0e933fc).